pkg/openstack/loadbalancer_test.go

```diff
@@ -534,3 +536,49 @@ func Test_getListenerProtocol(t *testing.T) {
 		})
 	}
 }
+
+func Test_getSecurityGroupName(t *testing.T) {
+	tests := []struct {
+		name     string
+		service  *corev1.Service
+		expected string
+	}{
+		{
+			name: "regular test security group name and length",
+			service: &corev1.Service{
+				ObjectMeta: v1.ObjectMeta{
+					UID:       "12345",
+					Namespace: "security-group-namespace",
+					Name:      "security-group-name",
+				},
+			},
+			expected: "lb-sg-12345-security-group-namespace-security-group-name",
+		},
+		{
+			name: "security group name longer than 255 byte",
```

Comment:
Can we name test more verbose? It's an ideal practice that a test name should also mention the ideal condition it had expected to be in.
 
```
security group name longer than 255 byte should get trimmed to 255 byte
```

Reply:
Yeah I understand you want to help but what the test does is check if it actually trimmed to 255 byte 

so i think it’s verbose enough that I am testing a security group name longer than 255 byte 

hence the name

Reply:
I agree, current name seems self-explanatory enough.